crates/store/re_grpc_server/src/main.rs

```diff
@@ -0,0 +1,22 @@
+use re_grpc_server::MessageProxy;
+use re_memory::MemoryLimit;
+use re_protos::sdk_comms::v0::message_proxy_server::MessageProxyServer;
+use tokio::net::TcpListener;
+use tonic::transport::server::TcpIncoming;
+use tonic::transport::Server;
+
+#[tokio::main(flavor = "current_thread")]
+async fn main() -> Result<(), tonic::transport::Error> {
+    let tcp_listener = TcpListener::bind("127.0.0.1:1852")
```

Comment:
This should use `re_log` to set up logging, and then log something useful on startup, like "Listening for gRPC connection on temp://…"